.changeset/tender-students-occur.md

```diff
@@ -0,0 +1,9 @@
+---
+'@pantheon-systems/nextjs-kit': minor
+'@pantheon-systems/wordpress-kit': minor
+---
+
+Modified the contentWithImage component on Next-Kit to remove the paddings and
+allow the content full page. Modified every component on the Wordpress-Kit
+tailwind plugin to respect the full width options. Added base utilities on
+tailwind plugin to modify headings and paragraphs width.
```

Comment:
Can we split this into 2 changests, one for each `nextjs-kit` and `wordpress-kit` so the generated changelog is clear for each package?